atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/AtlasDbConstraintCheckingMode.java

```diff
@@ -22,14 +22,14 @@
     PERFORMANCE_OPTIMAL_CONSTRAINT_CHECKING_LOGS_EXCEPTIONS(ShouldThrowException.NO),
     NO_CONSTRAINT_CHECKING(ShouldThrowException.NO);
 
-    private static enum ShouldThrowException {
+    private enum ShouldThrowException {
         YES,
         NO
     }
 
     private final ShouldThrowException shouldThrowException;
 
-    private AtlasDbConstraintCheckingMode(ShouldThrowException shouldThrowException) {
+    AtlasDbConstraintCheckingMode(ShouldThrowException shouldThrowException) {
```

Comment:
Redundant qualifier. Enum constructors can only be called within the enum itself.